test/libsolidity/semanticTests/tryCatch/assert.sol

```diff
@@ -0,0 +1,16 @@
+contract C {
+    function g(bool x) public pure {
+        assert(x);
+    }
+    function f(bool x) public returns (uint) {
+        // Set the gas to make this work on pre-byzantium VMs
+        try this.g.gas(8000)(x) {
+            return 1;
+        } catch {
+            return 2;
```

Comment:
return gas left instead to demonstrate that `assert` style exceptions consume gas? Perhaps add a symmetric test case for `require` that demonstrates no gas consumption,

Reply:
I think this is hard to get stable.